code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -22,6 +24,7 @@
 /obj/item/ego_weapon/examine(mob/user)
 	. = ..()
 	. += EgoAttackInfo(user)
+	. += "<span class='notice'>[special]</span>"
```

Comment:
It'd be a good idea to not display it at all if it's null, which it should be when it doesn't have special stuff.

Reply:
Good point, I just had it to set a "This has no special abilitieS" message to keep them all about 4 lines long